uninstall.sh

```diff
@@ -181,12 +191,16 @@ if ! uninstall_from_existing_repositories; then
     exit 1
 fi
 
+# Uninstall all shared hooks
+uninstall_shared_hooks
+
 # Unset global Githooks variables
 git config --global --unset githooks.shared
 git config --global --unset githooks.autoupdate.enabled
 git config --global --unset githooks.autoupdate.lastrun
 git config --global --unset githooks.previous.searchdir
 git config --global --unset githooks.disable
+git config --global --unset alias.hooks
```

Comment:
Technically, we could still use the cli with the alias to reinstall, though I think it was probably an oversight from me. If we unset the alias now, we should also delete the cli script.

Reply:
shouldnt we just completely delete the `~/.githooks` folder, without the potential `template` subfolder.. or also with it? because it anyways contains githooks templates and not own ones... hm...

Reply:
No I think its not good, since the template folder is choosen by the user, we leave it there, but delete all other stuff `bin,shared` folders...

Reply:
Unexcepted behavior if you have user defined hooks in `.githooks/templates` which then get correctly back replaced but at the end deleted, which is stupid...